ddtrace/_trace/span.py

```diff
@@ -260,6 +260,23 @@ def duration(self, value):
         # type: (float) -> None
         self.duration_ns = int(value * 1e9)
 
+    @property
+    def sampled(self):
+        # type: () -> bool
+        if not self.context.sampling_priority:
+            return False
+        return self.context.sampling_priority > 0
```

Comment:
We can simplify this a bit:
```suggestion
        return self.context.sampling_priority != None and self.context.sampling_priority > 0
```